packages/commuter-server/config.js

```diff
@@ -0,0 +1,14 @@
+const { bucket = null, pathDelimiter = "/", basePath = null } = process.env;
```

Comment:
Small nit: I'd probably make these upper case with `_` like `PATH_DELIMITER` as well as prefixed. `COMMUTER_BUCKET`, `COMMUTER_PATH_DELIMITER`, and `COMMUTER_BASEPATH`. It makes sysops folks much happier.